packages/igx-templates/igx-ts/custom-templates/crm-grid/files/src/app/__path__/__filePrefix__.component.html

```diff
@@ -32,16 +32,16 @@
 							</span>
 						</div>
 						<div class="caseSensitiveButton">
-							<button igxButton="icon" igxRipple igxRippleCentered="true" (click)="updateSearch()"
+							<button igxButton="icon" igxRipple [igxRippleCentered]="true" (click)="updateSearch()"
```

Comment:
No need for square brackets here and below as well as the value is set in the template.

Reply:
Changed to the suggested.